src/config/entities/configuration.ts

```diff
@@ -1,4 +1,15 @@
+import { readFileSync } from 'fs';
+
+const packageJson = JSON.parse(
+  readFileSync('package.json', { encoding: 'utf-8' }),
```

Comment:
I have some concerns regarding this approach. This basically makes the whole `package.json` available to the running instance – I think `package.json` should be seens a build/publishing tool and therefore we should avoid using it directly in the code (also ties the service to the build/publishing tool being used).

My suggestion is to find a more independent alternative to it. The version was previously retrieved with an environment variable – `npm_package_version` – which is only available while using `npm`. We can have a configuration script that sets the environment variable inside `pakage.json` and use that environment variable here.

Reply:
I agree the optimal solution would be setting the data into a environment variable. 

Maybe we can output the version and the name during the building phase and pass that variable to the production image. I'll try to do it with `yarn` or even with `npm`, both should be available in the build container. What do you think @fmrsabino ?

Reply:
> I agree the optimal solution would be setting the data into a environment variable.
> 
> Maybe we can output the version and the name during the building phase and pass that variable to the production image. I'll try to do it with `yarn` or even with `npm`, both should be available in the build container. What do you think @fmrsabino ?

Yes I prefer this approach. Maybe a script that just sets these environment variables in the current session. Then the service wouldn't depend on the structure of the `package.json` but only on the local environment.

Reply:
>An environment variable set on the build phase and also on Docker itself should be simpler.

Totally agree 👍 , but based on my findings I think I won't be easy, unless we maintain the version number outside `package.json` and set it as environment variable in the container.

> My suggestion is to focus only on the controller/route part for this.

You mean just returning an `null`/`` for the version in the controller for now? 

> Also I'd say that this doesn't need to be tackled right now and probably a separate issue for the versioning would be better.

Yes, my bad, I thought I had found a simpler solution but yes, let's tackle this in a separate PR when we reach a consensus about the approach.